src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -937,6 +941,8 @@ void Deconvolution::prepareParams() {
         } else {
             std::tie(desc, fwd_conv_pd) = createDefaultMkldnnDeconvDesc(key.inp0->getDnnlDesc(), key.inp1->getDnnlDesc(), key.out->getDnnlDesc(),
                                                                         key.stride, key.dilation, key.paddingL, key.paddingR, key.attr, engine);
+            // Create dummy primitive to WA CC issue.
+            OPENVINO_ASSERT(dnnl::primitive(fwd_conv_pd));
```

Comment:
Can we also put it under (please double check the correct define in this case)?
`#ifdef SELECTIVE_BUILD_ANALYZER`
So it has no impact on the regular flow?

Reply:
ok, it is better!